### PR TITLE
Set readonly_language to false for desktop roles (boo#1212560)

### PR DESCRIPTION
**Pull request by Vogtinator**
Desktops are multilingual, unlike the "plain" MicroOS roles.

### CODE REVIEW

**Review by shundhammer — APPROVED** [review]
LGTM

**Comment by shundhammer** [comment]
Oops; one thing that is missing:

Please bump the version in the .spec file and add a change log entry referring to the bug. TIA.

**Comment by Vogtinator** [comment]
Ah right, this is a YaST repo. Will do.

**Comment by Vogtinator** [comment]
Done. I'm a bit surprised that `version:bump` doesn't also add a .changes entry but `osc vc` works fine.

**Comment by sysrich** [comment]
As noted in https://bugzilla.opensuse.org/show_bug.cgi?id=1212560#c20 please can YaST reviewers like @shundhammer please take care to involve me or @sfalken on any changes that will effect the desktop roles

I am the lead maintainer of the GNOME desktop role, and Shawn the KDE one. While Shawn knew of this change coming towards his role, this was merged despite his objections.

I'm only finding out about it after the merging, which is not appreciated, and seems suboptimal.

**Comment by shundhammer** [comment]
Sigh... I really wish you guys would talk to each other. We are always just on the receiving end of conflicting demands. First we get a bug report that we have nothing to do with, then it turns out that it was due to an arcane setting that almost nobody knows, then we were asked to provide a workaround for a problem that we didn't create in the first place, then there is a change request, then we think we support our colleagues by approving their PRs.

**Pull request merged**
